chain-impl-mockchain/src/ledger/ledger.rs

```diff
@@ -353,14 +355,20 @@ impl Ledger {
 
         let epoch = new_ledger.date.epoch + 1;
 
-        let expected_epoch_reward =
-            rewards::rewards_contribution_calculation(epoch, &ledger_params.reward_params);
+        // changed to scale rewards calculation
+        let expected_epoch_reward = rewards::rewards_contribution_calculation_scaled(
+            epoch,
+            &ledger_params.reward_params,
+            // Pass total stake for all pools from distribution -- KH
+            distribution.total_stake(),
+        );
 
         let mut total_reward = new_ledger.pots.draw_reward(expected_epoch_reward);
 
+        // Changed from true to false -- rewards *must* accrue to the treasury in the testnet.   KH
```

Comment:
what's the meaning of `KH` ?

Reply:
Just an indicator to Jared and me that this is something I have changed.  Should be removed in the latest version.